Update Campaigns.php

JoseVega opened the pull request:
When using the create and update methods of the campaigns endpoint, the plain text content must be base64 encoded the same way as the html content, otherwise the HTTP request breaks and the API returns an "Invalid API request signature." error.

The mailwizz app expects the plain text content to be encoded ( see mailwizz/apps/api/controllers/campaigns.php line 236 ).

JoseVega committed:
Update Campaigns.php

The plain text content must be base64 encoded the same way as the html content, otherwise the HTTP request breaks and the API returns an "Invalid API request signature." error.

The mailwizz app expects the plain text content to be encoded ( see mailwizz/apps/api/controllers/campaigns.php line 236 ).

twisted1919 commented:
Nice catch Jose, if you wnat this merged, you'll have to also modify the update method, otherwise let me know and i'll simply update the code in my repo and push the changes.

Thanks.

JoseVega committed:
Update Campaigns.php

When using the update method of the campaigns endpoint, the plain text content must be base64 encoded the same way as the html content, otherwise the HTTP request breaks and the API returns an "Invalid API request signature." error.

The mailwizz app expects the plain text content to be encoded ( see mailwizz/apps/api/controllers/campaigns.php line 236 ).

JoseVega commented:
Your right, I just modified the update method.

twisted1919 commented:
Thanks, i have merged the changes.
